package.json

```
@@ -42,9 +42,11 @@
   "dependencies": {
     "emotion": "9.2.12",
     "emotion-theming": "9.2.9",
+    "focus-trap-react": "4.0.1",
     "memoize-one": "4.0.2",
     "react": "16.5.2",
     "react-click-outside": "3.0.1",
+    "react-delegate-component": "^1.0.0",
```

We normally use pinned versions

```suggestion
     "react-delegate-component": "1.0.0",
```

---

Plus the dependency has 4 commits and was not changed over a year. Do you think it will survive upcoming react updates or are we okay with maintaining a fork in the long run if the author remains inactive?